File: GoalCount/app/src/main/java/com/example/goalcount/CountEntry.java
package com.example.goalcount;

import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "count_entries")
public class CountEntry {
    @PrimaryKey(autoGenerate = true)
    public int id;
    public int count;
    public long timestamp;

    public CountEntry(int count, long timestamp) {
        this.count = count;
        this.timestamp = timestamp;
    }
}

File: Count/app/src/main/java/com/example/count/data/CountEntry.kt
package com.example.count.data

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "counts")
data class CountEntry(
    @PrimaryKey(autoGenerate = true)
    val id: Int = 0,
    val value: Int,
    val increment: Int,
    val timestamp: Long = System.currentTimeMillis()
)
